relayer/application.ts

```diff
@@ -147,7 +147,6 @@ export class RelayerApp<ContextT extends Context> extends EventEmitter {
       return false;
     }
     for (let i = 0; i < this.vaaFilters.length; i++) {
-      const { emitterChain, emitterAddress, sequence } = vaa.id;
```

Comment:
note: these variables were redundant. We already de-structured these values outside for-loop